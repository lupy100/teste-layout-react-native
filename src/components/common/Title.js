// @flow
import styled from 'styled-components/native';

export default styled.Text`
  font-size: 40;
  text-align: center;
  color: #ffffff;
`;
