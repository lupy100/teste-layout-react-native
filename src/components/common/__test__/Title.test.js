// @flow
import * as React from 'react';
import renderer from 'react-test-renderer';

import Title from '../Title';

test('renders Title correctly', () => {
  const tree = renderer.create(<Title>This is my title</Title>).toJSON();
  expect(tree).toMatchSnapshot();
});
