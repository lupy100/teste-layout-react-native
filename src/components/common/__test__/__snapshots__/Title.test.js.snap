// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#ffffff",
        "fontSize": 40,
        "textAlign": "center",
      },
      undefined,
    ]
  }
>
  This is my title
</Text>
`;
